FileManipulatorAbstractions/GlobalJsonManipulationException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Messerli.FileManipulatorAbstractions
+{
+    public sealed class GlobalJsonManipulationException : Exception
+    {
+        private readonly string _filePath;
+
+        public GlobalJsonManipulationException(Exception innerException, string filePath)
+            : base(null, innerException)
+        {
+            _filePath = filePath;
+        }
+
+        public override string Message => $"Error manipulating file {_filePath}";
```

Comment:
"... see inner exception for details"

Reply:
https://github.com/messerli-informatik-ag/meta-generator/pull/14/commits/d784197f2cd619c703a83ef4dc239092708bed4c